package/MDAnalysis/lib/distances.py

```diff
@@ -50,10 +50,56 @@
                                         with OpenMP
    ========== ========================= ======================================
 
+Use of the distopia library
+---------------------------
+
+MDAnalysis has developed a standalone library, `distopia`_ for accelerating
+the distance functions in this module using explicit SIMD vectorisation.
+This can provide many-fold speedups in calculating distances. Distopia is
+under active development and as such only a selection of functions in this
+module are covered. Consult the following table to see if the function
+you wish to use is covered by distopia. For more information see the
+`distopia documentation`_.
+
+.. Table:: Functions available using the `distopia`_ backend.
+
+    +-------------------------------------+
+    | Functions                           |
```

Comment:
If a summary is being provided, I would have a second column for extra details, like the fact that it doesn't work for triclinic boxes.